Mage/Module.cs

```diff
@@ -90,6 +90,11 @@ public class Module<T> : Singleton<T> where T : class, new() {
 	
 	public void setupUsercommands (Action<Exception> cb) {
 		logger.info ("Setting up usercommands");
+
+		if (commands == null) {
```

Comment:
Alright, forget about (a)sync for now. But my question remains: why do null and empty list behave differently?